styles/blocks/f-button-group.css

```diff
@@ -58,27 +36,29 @@ I think it would look awesome!
 			<button class="f-button-group__button_underline">Who we are</button>
 			<button class="f-button-group__button_underline">Dank memes</button>
 		</div>
-	</section>
-
-<!-- this should be used on the website -->
-</div>
 ```
-
-
 */
 
 @import "../variables";
 
-.f-button-group { 
+.f-button-group {}
 
+.f-button-group__button {
+	display: inline;
```

Comment:
@troldmand Since you are using `f-column` to layout the buttons, they are implicit *flex items*. A **flex item** can not be anything other than an **flex item** so the display property is ignored.
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Using_CSS_flexible_boxes